src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -838,6 +839,29 @@ def create(notebook_path: str | Path | None = None, **kwargs):
     yield from factory("job", create, lambda item: ws.jobs.delete(item.job_id))
 
 
+@pytest.fixture
+def make_ws_query(ws, make_random):
```

Comment:
there's `make_query` already.

Reply:
the existing is one is different, it also creates a table (which we don't want here), and populates the SQL arbitrairily which we also don't want.